### PR TITLE
fix(UiUtils): correctly lookup local account

### PR DESCRIPTION
**Pull request by FineFindus**
Fixes a regression in f590fde7a437ae17d1858bb24191bed04491fb79, where links to local accounts would be opened in the browser.

### CODE REVIEW

File: mastodon/src/main/java/org/joinmastodon/android/ui/utils/UiUtils.java [hunk 1]
```diff
@@ -1341,10 +1341,6 @@ public static void openURL(Context context, String accountID, String url) {
 		openURL(context, accountID, url, true);
 	}
 
-	public static void openURL(Context context, String accountID, String url, Object parentObject) {
```
Comment (LucasGGamerM) [inline mastodon/src/main/java/org/joinmastodon/android/ui/utils/UiUtils.java:1344]: Why did you remove this? I think it was uptream that added it, so maybe we should keep it just in case.

Comment (FineFindus) [reply mastodon/src/main/java/org/joinmastodon/android/ui/utils/UiUtils.java:1344]: It's not used anywhere and hasn't been for awhile

Comment (LucasGGamerM) [reply mastodon/src/main/java/org/joinmastodon/android/ui/utils/UiUtils.java:1344]: Sure... In case upstream needs it we will readd it. But this should be 2 commits instead of one.

**Review by LucasGGamerM — APPROVED** [review]
I may or may not have forgotten about this :)
